sdk/trace/build.gradle.kts

```diff
@@ -45,6 +45,7 @@ dependencies {
     jmh("io.grpc:grpc-api")
     jmh("io.grpc:grpc-netty-shaded")
     jmh("org.testcontainers:testcontainers") // testContainer for OTLP collector
+    implementation("org.jctools:jctools-core:3.2.0")
```

Comment:
We do not want to take additional external dependencies like this for our SDK.

Reply:
If we want to use this, we'll need to shade it into our project, rather than have it be exposed as a transitive dependency.